### DISCUSSION TITLE
온라인 마케팅의 핵심, UTM이 무엇일까?

### DISCUSSION THREAD

**Discussion by giscus** [yiyb0603/yiyb-blog#66] [Blog Comment]
# 온라인 마케팅의 핵심, UTM이 무엇일까?

온라인 마케팅의 효과를 분석, 추적할 수 있도록 도와주는 UTM 파라미터에 대해서 알아봅시다.

https://yiyb-blog.vercel.app/posts/utm-parameters

**Comment by DreamYunha** [comment]
오늘 처음 알게된 정보입니다. 감사해요!

**Reply by yiyb0603** [reply]
도움이 되어서 다행입니다. 글 읽어주셔서 감사합니다!
